projects/novo-elements/src/elements/form/controls/BaseControl.ts

```diff
@@ -97,6 +97,7 @@ class ControlConfig {
   isInlineEmbedded = false;
   weekStart?: number;
   highlighted = false;
+  customTooltip?: string;
```

Comment:
Expanding this list on the base control affects everything, and will need support in the Field Interaction API for completeness. I suggest not adding a customTooltip, but instead reusing tooltip when we need a disabled tooltip, since tooltip is not currently used by the date picker.